src/browser/web-view.js

```diff
@@ -260,7 +260,11 @@ define((require, exports, module) => {
       onTitleChange: event => edit(WebView.setTitle(event.detail)),
       onPrompt: event => console.log(event),
       onAuthentificate: event => console.log(event),
-      onScrollAreaChange: state.contentOverflows == void(0) && (event =>
+      // This will trigger a resize. If the content react to the resize by changing its
+      // layout, this might change the scrollarea again, triggering a resize… infinite
+      // loop.
+      // So we only allow contentOverflows to transition from false (default value) to true.
+      onScrollAreaChange: !state.contentOverflows && (event =>
```

Comment:
I tried this approach first and observed odd behavior on gmail.com and few other sites, I was getting double scrolling or something strange like that. I'm unable to test this right now, but please test on gmail.com and see if that really a case we should probably not land this change.